packages/integration-karma/test/synthetic-shadow/inner-outer-text/inner-outer-text.spec.js

```diff
@@ -2,21 +2,12 @@ import { createElement, setFeatureFlagForTest } from 'lwc';
 import Container from 'x/container';
 
 if (!process.env.NATIVE_SHADOW) {
-    describe('DISABLE_INNER_OUTER_TEXT_PATCH flag', () => {
-        const elm = createElement('x-container', { is: Container });
-        document.body.appendChild(elm);
-
-        it('should get innerText of custom element when DISABLE_INNER_OUTER_TEXT_PATCH = true', () => {
-            setFeatureFlagForTest('DISABLE_INNER_OUTER_TEXT_PATCH', true);
-            expect(elm.innerText).not.toBe('');
-            setFeatureFlagForTest('DISABLE_INNER_OUTER_TEXT_PATCH', false);
-        });
+    beforeAll(() => {
+        setFeatureFlagForTest('ENABLE_INNER_OUTER_TEXT_PATCH', true);
+    });
 
-        it('should get outerText of custom element when DISABLE_INNER_OUTER_TEXT_PATCH = true', () => {
-            setFeatureFlagForTest('DISABLE_INNER_OUTER_TEXT_PATCH', true);
-            expect(elm.outerText).not.toBe('');
-            setFeatureFlagForTest('DISABLE_INNER_OUTER_TEXT_PATCH', false);
-        });
```

Comment:
Not sure why we should remove the test?

Reply:
these tests were for the feature flag itself